packages/gatsby-theme-api-docs/src/components/resource/method/request-response-examples.js

```diff
@@ -46,13 +48,9 @@ const RequestResponseExamples = (props) => {
       {responsesCodeExamples.map((codeExample) => {
```

Comment:
since we're here: can we rely on the responses being ordered numerically by code?